release-engineering/platforms.md

```diff
@@ -0,0 +1,67 @@
+# Supported Platforms
+
+The Kubernetes release process produces [artifacts](./artifacts.md) for
+different architectures and operating systems. We consider the combination of
+architecture (`GOARCH`) and operating system (`GOOS`) as "platforms". Target of
+this document is to outline different categories of platforms as well as guiding
+through their graduation criteria.
+
+## Tiers
+
+Support for different platforms are organized into three tiers, whereas each
```

Comment:
suggest "Build and release support for different platforms' artifacts are..."

Reply:
Changed as suggested :+1: